### PR TITLE
add tooltips and update show_templates

### PR DESCRIPTION
**Pull request by oleg08**
code review:
- [x] @DimaSmagulov 
- [ ] @krisberry 
- [x] @natalya5 
- [ ] @CHILL99 
- [x] @OrestF 
- [x] @rtershak 
- [x] @vvs79 

2nd lvl review:

- [x] @Kmalynss
- [ ] @serhiym 
- [ ] @ruba-ruba

### CODE REVIEW

**Comment by OrestF** [comment]
reviewed

**Comment by vvs79** [comment]
reviewed

**Comment by rtershak** [comment]
reviewed

**Comment by natalya5** [comment]
reviewed

File: app/views/templates/_template.html.slim [hunk 1]
```diff
@@ -1,6 +1,6 @@
 tr.info
   td
-    =link_to template_path(template)
+    =link_to template_path(template), rel: 'tooltip', title: 'Show template'
       = template.name
   td
     .btn-toolbar aria-label="..." role="toolbar"
```
Comment (DimaSmagulov) [inline app/views/templates/_template.html.slim]: pls make the same buttons like in _question.html.slim and give them tooltips too.

Comment (DimaSmagulov) [inline app/views/templates/_template.html.slim]: pls make the same buttons like in _question.html.slim in this toolbar and give them tooltips too.

Comment (Kmalynss) [inline app/views/templates/_template.html.slim]: @oleg08 did you address @DimaSmagulov comment?

Comment (oleg08) [inline app/views/templates/_template.html.slim]: Yes, I addressed @DimaSmagulov comment

**Pull request merged**